.circleci/config.yml

```diff
@@ -75,6 +75,7 @@ references:
                 rosdep install -q -y \
                   --from-paths src \
                   --ignore-src \
+                  --skip-keys slam_toolbox \
```

Comment:
Try and use the same quotation and new line arrangement from the commit I linked before such that each key is on a separate line, so we can easily track the addition and removal of different rosdep keys in revision control.

Reply:
@ruffsl the test results are certainly... interesting. Looks like one of the mixins is old, though I'm not entirely sure what that was added to support

Reply:
I originally passed the same mixin arguments to both colcon build and test so one doesn't need to specify so many docker build args, but it looks like cocaine is switched to raise an error rather than only a warning for passing unused mixins. We're not using those mixins for colcon test currently anyway, so that could be stripped out there.

Reply:
Just to verify, we can just remove them? Are you aware why they were added?